flink-python/MANIFEST.in

```diff
@@ -29,3 +29,4 @@ include README.md
 include pyflink/LICENSE
 include pyflink/NOTICE
 include pyflink/README.txt
+graft pyflink/proto
```

Comment:
It seems we don't need this change. The proto dir has been included in the pyflink.zip?